Makefile

```diff
@@ -22,7 +22,7 @@ export KUBEBUILDER_CONTROLPLANE_START_TIMEOUT ?=60s
 export KUBEBUILDER_CONTROLPLANE_STOP_TIMEOUT ?=60s
 
 # Image URL to use all building/pushing image targets
-export CONTROLLER_IMG ?= gcr.io/k8s-cluster-api/cluster-api-controller:latest
+export CONTROLLER_IMG ?= gcr.io/k8s-cluster-api/cluster-api-controller:0.1.0
 export EXAMPLE_PROVIDER_IMG ?= gcr.io/k8s-cluster-api/example-provider-controller:latest
```

Comment:
Note that I am _not_ changing the EXAMPLE_PROVIDER_IMG ... I figure that's for CI and we should not release it (?)

Reply:
I think that correct

/cc @figo

Reply:
CI will set the EXAMPLE_PROVIDER_IMG based on git hash at runtime. so the tag `latest` here does not matter, hence no need to change.

Reply:
Should we prepend the version with a v (note I'm not overly opinionated either way)?